### PR TITLE
[ADD] new "hr_rule_input_compute" module

### PR DESCRIPTION
**Pull request by ufaks**

### CODE REVIEW

**Comment by ilmir-k** [comment]
@ufaks 
runbot errors need to be fixed

File: hr_rule_input_compute/doc/index.rst [hunk 1]
```diff
@@ -0,0 +1,27 @@
+=======================
+ Compute Salary Inputs
+=======================
+
+The module allows to make salary rule inputs computable. A ``Python Code`` field in a salary rule form is editable for users with a ``group_hr_rule_input_compute`` group. The users can write code which computes amounts of salary rule inputs, e.g.::
```
Comment (yelizariev) [inline hr_rule_input_compute/doc/index.rst]: Replace group_hr_rule_input_compute with non-technical name of the group

File: hr_rule_input_compute/security/hr_rule_input_compute_security.xml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="0">
+
+        <record id="group_hr_rule_input_compute" model="res.groups">
+            <field name="name">Make editable a field for computing salary rule inputs</field>
```
Comment (yelizariev) [inline hr_rule_input_compute/security/hr_rule_input_compute_security.xml]: <field name="name">Can edit "Python Code" field for salary inputs</field>

**Review by yelizariev — COMMENTED** [review]
Need some updates with description

**Pull request merged**